site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1530,6 +1558,24 @@ components:
                     format: byte
                     description: The secret key shared between device and Hono.
 
+      RPKSecret:
+         additionalProperties: false
+         allOf:
+            - $ref: '#/components/schemas/CommonSecret'
+            - type: object
+              additionalProperties: false
+              required:
+                 - algorithm
+                 - key
+              properties:
+                 "algorithm":
+                    type: string
+                    description: The signing algorithm of the provided key (supported algorithms are `RSA` and `EC`).
+                 "key":
+                    type: string
+                    description: |
+                       The Base64 encoded bytes representing the public key / certificate.
```

Comment:
Again, this can only hold the public key, not a certificate,
Thus, this should read
> The Base64 encoded binary DER encoding of the public key.

in order to be very precise

Reply:
ok I think I got it now and changed it accordingly